rascal-vscode-extension/src/test/vscode-suite/dsl.test.ts

```diff
@@ -166,7 +166,7 @@ describe('DSL', function () {
 
         // finds an open menu with the right item in it (Change to a) and then select
         // the parent that handles UI events like click() and sendKeys()
-        const menuContainer = await ide.hasElement(editor, By.xpath("//div[contains(@class, 'focused') and contains(@class, 'action')]/span[contains(text(), 'Change to a')]//ancestor::*[contains(@class, 'monaco-list')]"), Delays.normal,"The Change to a option should be available and focussed by default");
+        const menuContainer = await ide.hasElement(editor, By.xpath("//div[contains(@class, 'focused') and contains(@class, 'action')]/span[contains(text(), 'Change to a')]//ancestor::*[contains(@class, 'monaco-list')]"), Delays.verySlow,"The Change to a option should be available and focussed by default");
```

Comment:
Do we still need this longer delay, even with the improved "find element" implementation below?

Reply:
I hope not, but it can't hurt. It will only take longer to crash in case it doesn't work. While if it's there, it will succeed within a second.